### PR TITLE
Move to the new all-in for the docker dev approach

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Comment by ericof** [comment]
@sneridagh Fixed acceptance tests.

**Review by ericof — APPROVED** [review]
Add a changelog entry and we are good to go

File: dockerfiles/Dockerfile.dev [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+ARG ADDON_NAME
+ARG ADDON_PATH
+
+COPY --chown=node:node package.json /app/src/addons/${ADDON_PATH}/
+
+RUN <<EOT
+    /setupAddon
+    yarn install --network-timeout 1000000
+EOT
```
Comment (davisagli) [inline dockerfiles/Dockerfile.dev:13]: @sneridagh @ericof Dockerfile.dev is identical to Dockerfile.acceptance. What's the reasoning for 2 different Dockerfiles?

**Review by davisagli — COMMENTED** [review]
Nice work.

**Pull request merged**